vendor/engines/saas/app/views/saas/trials/show.html.erb

```diff
@@ -4,7 +4,7 @@
     <div class="trial_form">
       <%= form_tag("/settings/profile/#{@user}/trial/activate", method: :post) do %>
         <input type="email" placeholder="Email Address" name="billing_email" required>
-        <input class="btn primary" type="submit" value="Do it!"/>
+        <input class="btn primary" type="submit" value="Do it!" data-analytics-path="/settings/<%=@user%>/trial/activate"/>
```

Comment:
Should this be `/settings/#{owner}/trialing` rather than `/trial/activate`? Or are you planning to update the funnel?